### PR TITLE
dropped support for sending lux binary

### PR DESCRIPTION
**Pull request by magdasta**

### CODE REVIEW

File: gnr/task/luxrendertask.py [hunk 1]
```diff
@@ -66,18 +66,17 @@ def __init__(self):
...
         self.halttime = 600
         self.haltspp = 1
-        self.send_binaries = False
         self.luxconsole = self.environment.get_lux_console()
 
-    def add_to_resources(self, resources):
-        if self.send_binaries and os.path.isfile(self.luxconsole):
-            resources.add(os.path.normpath(self.luxconsole))
-        return resources
+    #def add_to_resources(self, resources):
```
Comment (chfast) [inline gnr/task/luxrendertask.py]: Remove code if not needed.

Comment (magdasta) [inline gnr/task/luxrendertask.py]: Thanks, forgot to delete this.

**Comment by magdasta** [comment]
Can I merge?

**Comment by chfast** [comment]
Yes, please.

**Pull request merged**